backend/parse.js

```diff
@@ -31,11 +31,28 @@ export function parseSQL(data) {
 
     // iterate over each object (each object becomes a row)
     // returns: a row of rows (table)
-    const rows = data.map(row =>
-        Object.keys(row).map(
-            columnname => row[columnname]
-        )
-    );
+    const rows = [];
+    for (let i = 0; i < data.length; i++) {
+        rows[i] = [];
+        for (let j = 0; j < columnnames.length; j++) {
+            let cell = data[i][columnnames[j]];
+            // Like PostGIS's GeoJSON or Dates
+            const cellType = type(cell);
+            if (contains(cellType, ['Object', 'Boolean'])) {
```

Comment:
I think redshift dates come out as `Object` types not as `Dates`, I can't connect to it right now to see if I am right. Need to check this to make sure and need to open up that redshift database.